protocol-units/bridge/sources/METHBridge.move

```diff
@@ -0,0 +1,455 @@
+module 0x1::METHBridge {
+    use std::signer;
+    use std::vector;
+    use aptos_framework::coin;
+    use aptos_framework::event;
+    use aptos_framework::aptos_coin::AptosCoin;
+    use aptos_framework::event::{EventHandle};
+    use aptos_framework::account;
+
+    const BRIDGE_ACCOUNT: address = @0x1; //Should change this later to actual bridge address
```

Comment:
Changing this to anything but `0x1` breaks the tests.

We can also think about having this pulled out into `Move.toml` as an address as `bridge_account` and remove this constant and use `@bridge_account` potentially